.github/actions/update-on-new-ipfs-tag/entrypoint.sh

```diff
@@ -25,6 +25,15 @@ else
      make
      http-api-docs > $API_FILE
 
+     # update installation docs
+     cd $ROOT # go back to root of ipfs-docs repo
+     CURRENT_IPFS_NUMBER=${CURRENT_IPFS_TAG:1}
+     LATEST_IPFS_NUMBER=${LATEST_IPFS_TAG:1}
+     while read -r file; do
+          echo "replacing $CURRENT_IPFS_NUMBER with $LATEST_IPFS_NUMBER in $file"
+          sed -E -i "s/$CURRENT_IPFS_NUMBER/$LATEST_IPFS_NUMBER/g" $file
+     done <<< "$(grep "current-ipfs-version" ./docs -R --files-with-matches)"
```

Comment:
@laurentsenta  turns out when  `<<<` is used we need to make sure we use `bash` and not `sh` – see fix in https://github.com/ipfs/ipfs-docs/pull/1121